src/main/java/com/faforever/client/mapstruct/ReplayMapper.java

```diff
@@ -136,8 +145,8 @@ default HashMap<String, List<GamePlayerStats>> mapTeamStatsFromParser(ReplayData
                                                                               null);
         Float factionFloat = (Float) armyInfo.get("Faction");
         Faction faction = Faction.fromFaValue(factionFloat.intValue());
-        GamePlayerStats stats = new GamePlayerStats(player, (byte) 0, team, faction, null,
-                                                    List.of(ratingJournal));
+        GamePlayerStats stats = new GamePlayerStats(false, faction, (byte) 0, team, (byte) 0, (byte) 0, null,
```

Comment:
What is the purpose of the `mapTeamStatsFromParser` function? I noticed that the GamePlayerStats only used some of the fields that were provided by the api, so I expanded it, but this function creates a GamePlayerStats object and feeds it data from a parsed replay, but I don't know which of the additional fields can be extracted from the parser.

Reply:
It is exactly as you say that it populates the team stats for local replay files. Local replays don't have all the data from the api so only fill in some fields

Reply:
so how should I deal with this situation? Put in some dummy data like I did here?

Reply:
Yeah that is fine